doc/dev/issues/resolve_issues_effectively.md

```diff
@@ -23,6 +25,23 @@ If you can provide detailed reproduction steps, it will help us locate and solve
 
 <hr/>
 
+## Summary Of Common Errors
+
+### Error from Track1 to Track2
+If the code you use needs to set wait() function for the Long Running Operation to wait for the result, it should be the SDK of Track1. We have stopped maintenance at present.
+
+We strongly recommend that you update the SDK version. Then you will find that in the SDK of Track2, we all use the function name prefixed with `begin_` for LRO operations, and the result() method can be used to get the returned result.
+
+### Error from Service
+Since the python SDK is generated based on the [rest API][rest API], it will not deliberately change the returned results. So if you have any questions about the response result value of the request, please open the issue under the [rest issue][rest issue].
```

Comment:
`So if you have any questions about the response result value of the request,`
it is better:
`So if you think the returned result is not as expected,`